### PR TITLE
Add 1.1.2 addresses for kovan

### PR DESCRIPTION
**Pull request by gbalabasquer**
I've linked the latest to 1.1.2. I'll keep doing that until we get it renamed to `active` or `current`

### CODE REVIEW

**Review by kmbarry1 — DISMISSED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -65,7 +65,8 @@ <h2>Release Calendar</h2>
 				</td>
 				<td>
 					<ul>
-						<li><a href="/releases/kovan/1.1.1/index.html">Release 1.1.1</a> (Friday, 04.09.2020)</li>						
+						<li><a href="/releases/kovan/1.1.1/index.html">Release 1.1.2</a> (Thursday, 24.09.2020)</li>
```
Comment (zdumitru) [inline index.html:68]: @gbalabasquer The link says it's 1.1.1

**Review by zdumitru — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (gbalabasquer) [reply index.html]: thanks, fixed

**Review by zdumitru — APPROVED** [review]

**Comment by gbalabasquer** [comment]
Closing this PR as we should have the mainnet addresses today, so doing both in the same branch.

**Pull request closed without merging**